packages/@lwc/babel-plugin-component/src/dynamic-imports.js

```diff
@@ -0,0 +1,65 @@
+const moduleImports = require('@babel/helper-module-imports');
+const { generateError } = require('./utils');
+const { LWCClassErrors } = require('@lwc/errors');
+
+function getImportSource(path) {
+    return path.parentPath.get('arguments.0');
+}
+
+function validateImport(sourcePath) {
+    if (!sourcePath.isStringLiteral()) {
+        throw generateError(sourcePath, {
+            errorInfo: LWCClassErrors.INVALID_DYNAMIC_IMPORT_SOURCE_STRICT,
+            messageArgs: [String(sourcePath)],
+        });
+    }
+}
+/*
+ * Expected API for this plugin:
+ * { dynamicImports: { loader: string, strict: boolean } }
+ */
+module.exports = function() {
+    let loaderRef;
```

Comment:
@pmdartus I think this might fail when compiling multiple files?
Will probably have to move it somewhere else.